system/posix/retcode/retcode_test.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright 2022 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package retcode_test
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"knative.dev/pkg/system/posix/retcode"
+)
+
+var (
+	errExampleError = errors.New("example error")
+)
+
+func TestCalc(t *testing.T) {
+	cases := testCases()
```

Comment:
Why not inline this declaration ([as we do elsewhere](https://github.com/knative/pkg/blob/main/apis/url_test.go#L28))?

```golang

cases := []struct {
  name string
  err error
  want int
}{{
  name: "nil",
  err: nil,
  want: 0,
}, {
  ...
}}